core/templates/pages/exploration-editor-page/exploration-editor-page.module.ts

```diff
@@ -73,10 +69,6 @@ import { RevertExplorationModalComponent } from './history-tab/modal-templates/r
   ],
   declarations: [
     CkEditorCopyToolbarComponent,
-    DeleteHintModalComponent,
-    DeleteInteractionModalComponent,
-    DeleteLastHintModalComponent,
-    DeleteSolutionModalComponent,
```

Comment:
Why these are moved to shared components module?

Reply:
@JeeveshGarg made PR #14454 in which he introduced some issues. which are shown in an issue made by me #14566. @JeeveshGarg  made another PR fixing those issue i.e. #14522. as I need this changes in mine PR so I fetched his PR and merged in my PR. `he added this component in the wrong file these components should be on shared-component-module so PR  #14522 is fixing them` @JeeveshGarg @ashutoshc8101 PTAL

Reply:
@ashutoshc8101 we already import SharedComponentsModule in exploration-editor-page, so no need to adding them here. In one of my merge PR, i add them by mistake.